Dockerfile.desktop

```diff
@@ -11,8 +11,8 @@ ENV DEBIAN_FRONTEND=noninteractive \
     MOZ_HEADLESS=1 \
     LC_ALL=C.UTF-8 \
     LANG=C.UTF-8 \
-    FIREFOX_VERSION=112.0 \
-    GECKODRIVER_VERSION=0.33.0
+    FIREFOX_VERSION=120.0 \
```

Comment:
Q: Why 120? https://whattrainisitnow.com/
- **ESR:** 115
- **Release:** 122 &mdash; _Firefox 123 ships on February 20 (in 11 days)_
- **Beta:** 123
- **Nightly:** 124

Reply:
This doesn't matter too much as there is a script to update it to the latest version on line 48